tools/topology/topology2/include/components/eqfir/passthrough.conf

```diff
@@ -0,0 +1,14 @@
+Object.Base.data."fir_eq" {
```

Comment:
why is this called passthrough.conf?

Reply:
so it is a flat response, does nothing - so "passthrough"

Reply:
It's actually a blob that sets every channel filter index (preset) to -1. It instructs the EQ to pass that channel directly to output. When every channel is such the process() happens when mem copy operations for entire period.